wildfly-builder-image/tests/features/basic.feature

```diff
@@ -108,6 +108,13 @@ Feature: Wildfly basic tests
     Then container log should contain WFLYSRV0025
     Then XML file /opt/server/standalone/configuration/standalone.xml should contain value 1000 on XPath //*[local-name()='socket-binding-group']/@port-offset
 
+  Scenario: Port offset in galleon provisioned configuration using overriding env var
+    When container integ- is started with env
+       | variable                                            | value           |
+       | SOCKET_BINDING_GROUP_STANDARD_SOCKETS__PORT_OFFSET  | 1000            |
+    Then container log should contain WFLYSRV0025
+    Then XML file /opt/server/standalone/configuration/standalone.xml should contain value 1000 on XPath //*[local-name()='socket-binding-group']/@port-offset
```

Comment:
When using the env variable, the value of the port-offset is not persisted, we can't check the content of the XML file.
I see another issue, it is advised to use PORT_OFFSET env variable, the CLI that shutdown the server during shutdown knows about this env variable and adjusts the management port. When using SOCKET_BINDING_GROUP_STANDARD_SOCKETS__PORT_OFFSET, the CLI fails to connect to the server to shutdown it.